_dashboards/dashboard/index.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: default
+title: Creating dashboards
+nav_order: 30
+has_children: true
+---
+
+# Creating dashboards
+
+The **Dashboard** application in OpenSearch Dashboards lets you visually represent your analytical, operational, and strategic data to help users quickly understand the current situation, giving users a high-level view of key metrics, simplifying data exploration, and delivering insights to users when and where they need them.
+
+In this tutorial you'll learn the basics of creating a dashboard using the **Dashboard** application and OpenSearch sample data. The sample dataset has existing sample visualizations, and you can use those visualizations or create new visualizations for the dashboard. For this tutorial, you'll do both. Once you've completed this tutorial, you'll have learned the foundations of creating a new dashboard with multiple panels in OpenSearch Dashboards. 
+
+This dashboard example (https://playground.opensearch.org/app/dashboards#/view/722b74f0-b882-11e8-a6d9-e546fe2bba5f?_g=(filters:!(),refreshInterval:(pause:!f,value:900000),time:(from:now-7d,to:now))&_a=(description:'Analyze%20mock%20eCommerce%20orders%20and%20revenue',filters:!(),fullScreenMode:!f,options:(hidePanelTitles:!f,useMargins:!t),query:(language:kuery,query:''),timeRestore:!t,title:'%5BeCommerce%5D%20Revenue%20Dashboard',viewMode:view) shows you what's possible with OpenSearch Dashboards.
+{: .note}
+
+## Getting familiar with user interface
+
+Before getting started, let's get familiar with the **Dashboard** user interface. The user interface comprises the following main parts:
+
+![Dashboard user interface]({{site.url}}{{site.baseurl}}/images/dashboards/dashboard-UI.png)
+
+- The **navigation panel** (A) on the left contains the OpenSearch Dashboards applications.
+- The **search** bar (B) lets you search for documents and other objects and add filters.
+- The **filter** (C) lets you apply different data perspectives to narrow a dashboard's results.
+- The **toolbar** (D) contains frequently used commands and shortcuts.
+- The **time filter** (E) lets you customize the time and date.
+- The **panel** (F) allows you to add existing visualizations to the dashboard or create new ones for the dashboard.
+
+## Definitions
+
+Learn some useful terminology for working with OpenSearch Dashboards and the **Dashboard** application.
+
+- _Dashboards_ is the abbreviated proper name for OpenSearch Dashboards. OpenSearch Dashboards is an open-source visualization tool designed to work with OpenSearch.
+- _Dashboard_ is the application in OpenSearch Dashboards that is used to track, analyze, and display data at a glance.
+- _dashboard_ or _dashboards_ refer to common names for a tool used to visually display data.
```

Comment:
should this be "refers" ?

Reply:
how about _dashboard_ and _dashboards_ are common names for the tools used...